### DISCUSSION TITLE
PrimeNG 18 Beta: What's Next for PrimeBlocks, Templates, and PrimeFlex?

### DISCUSSION THREAD

**Discussion by SparrowVic** [primefaces/community#2769] [PrimeNG]
Hey PrimeTek,
Just wanted to say I'm really excited about the PrimeNG 18 beta release. The new theming system pleasantly surprised me, and I can't wait to see what's next!
I wanted to share what I currently understand about the changes in version 18 and seek clarification on a few points:

- Tailwind CSS Integration: I see that PrimeNG 18 is integrating with Tailwind CSS and moving away from PrimeFlex. This seems promising for flexibility and customization.
- Future of PrimeFlex: Will PrimeFlex continue to be developed, or are you planning to fully switch over to Tailwind CSS?
- PrimeBlocks and Templates: With the move away from PrimeFlex, I'm curious about the future of PrimeBlocks and PrimeTemplates. Will there be new blocks and templates (including premium ones) adapted to Tailwind CSS, or do you have another plan? When? :D 

Any info would be greatly appreciated! Knowing your plans will help many users better prepare for projects and keep making the most of your awesome tools.
Thanks for all your hard work!

**Comment by qwadrox** [comment]
Hi @SparrowVic 

I can answer your questions because it's written in the docs: 

https://v18.primeng.org/guides/primeflex

> "the team has decided to avoid the maintenance of a CSS library (PrimeFlex) and utilize the resources on improving the UI libraries instead"

> "PrimeTek officially suggests Tailwind CSS as the replacement for PrimeFlex"

And yes, there are going to be new blocks and templates as well, and they have already shipped the Vue.js version: https://primeblocks.org/
(Angular is coming soon as well.)

I recommend you follow him to be updated with the latest news: [Çağatay Çivici](https://www.linkedin.com/in/%C3%A7a%C4%9Fatay-%C3%A7ivici-1579b0129/overlay/about-this-profile/)
